integrations/tensorflow/e2e/CMakeLists.txt

```diff
@@ -96,3 +96,51 @@ iree_e2e_cartesian_product_test_suite(
 )
 
 ### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
+
+iree_python_benchmark_suite(
+  MODELS
+    MobileBERT
+  MODEL_SCRIPT
+    mobile_bert_squad_benchmark.py
+  TARGET_BACKEND
+    iree_vulkan
+  CONFIGURATION_NAME
+    "GPU-Mali-G77"
+  COMPILATION_FLAGS
+    "--iree-vulkan-target-triple=valhall-g77-unknown-android10"
```

Comment:
I'd rather see generic configurations in the main code tree, with individual phone models specified in a buildkite/CI path. Developers (members of the team and external users) should be able to run our collection of benchmarks on their own hardware easily. It should also be easy for us to change which devices we run benchmarks on.

Reply:
Yeah. The triple and the configuration name is a bit misleading here. It should be "GPU-Mali-Valhall" (which is the microarchitecture) and it can map to a lot of phones with Mali GPUs having this uArch. There should be another step (with its own build rules and scripts) to query a phone's CPU/GPU arch and then decide which suitable benchmark suite to run on it.